README_CURRENT_ISSUES.scd

```diff
@@ -0,0 +1,15 @@
+
+/*
+
+current issues
+
+(what you should know if you didn't follow the messages passed around)
+
+edit this file as needed to communicate the state of affairs.
+
+*/
+
+
+3.7 is basically done now.
```

Comment:
this is a bit imprecise. it makes it sound like work is mostly finished and it should be released soon.  but what you mean is: we are doing maintenance and major bug fixes on it.  we have to discuss those things a bit anyway